index.html

```diff
@@ -10,6 +10,11 @@
       <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
   </head>
   <body>
+    <div class="burger">
+      <div class="bar"></div>
+      <div class="bar"></div>
+      <div class="bar"></div>
+    </div>
     <div id="menu">
       <div id="about" class="section">
```

Comment:
Both of your divs with the class of section should be section tags for better semantic markup